README.md

```diff
@@ -25,6 +25,19 @@ cd bug-issue-ui
 yarn
 ```
 
+### Environment variables
+
+To get started local development, create a file in the root of the repo named
+`.env` with the following content.
+
+```bash
+GITHUB_PERSONAL_ACCESS_TOKEN = <insert personal access token here>
+```
+
+To get a github personal access token, go to [Github Settings Page](https://github.com/settings/profile),
+click on the Developer Settings, then click on Personal Access Token, and click on Generate New Token.
+For the scopes, select all scopes under `repo` category. After that, click Generate Token and copy the token to the above.
```

Comment:
How about:

Generate a personal token [here](https://github.com/settings/tokens). When prompted about scopes, **access to public repositories** is the only one required.